### PR TITLE
Fix uncached method.

### PR DESCRIPTION
**Pull request by robertzk**
Addresses https://github.com/robertzk/cachemeifyoucan/issues/75

### CODE REVIEW

**Comment by peterhurford** [comment]
Much closer to what I expected. :+1:

**Comment by peterhurford** [comment]
@robertzk You should add test coverage?

**Comment by robertzk** [comment]
Since @kirillseva did not comment, I am merging.

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,8 @@ before_script:
 install:
 - "./travis-tool.sh install_deps"
 - "sudo R CMD javareconf"
-- "./travis-tool.sh install_github rstats-db/RPostgres"
+- "./travis-tool.sh install_github rstats-db/DBI rstats-db/RPostgres"
+Fix DBI
```
Comment (kirillseva) [inline .travis.yml:15]: remove this line, your vim magic failed

Comment (peterhurford) [inline .travis.yml:15]: Need more [send.vim](github.com/peterhurford/send.vim).

**Pull request merged**